app/components/AppFooter.vue

```diff
@@ -20,6 +21,7 @@ const socialLinks = computed(() => {
     .filter(Boolean)
 })
 
+const iconLogo = '/icon.svg'
 const unjsLogo = '/unjs.svg'
```

Comment:
can we use "https://unjs.io/favicon.svg" ?

Reply:
Yes, no reason to not use it.